pkg/provisioner/ironic/ironic.go

```diff
@@ -321,6 +321,52 @@ func (p *ironicProvisioner) getNode() (*nodes.Node, error) {
 	}
 }
 
+// Verifies that node has port assigned by Ironic.
+func (p *ironicProvisioner) nodeHasAssignedPort(ironicNode *nodes.Node) bool {
+	opts := ports.ListOpts{
+		Fields: []string{"node_uuid"},
+	}
+
+	opts.NodeUUID = ironicNode.UUID
```

Comment:
nit: why is this not being set in the initialiser above?

Reply:
yea, missed that, thanks, fixed